cmd/migrate_from_milvus.go

```diff
@@ -76,7 +76,7 @@ func (r *MigrateFromMilvusCmd) Run(globals *Globals) error {
 		return fmt.Errorf("failed to connect to Qdrant target: %w", err)
 	}
 
-	err = commons.PrepareMigrationOffsetsCollection(ctx, r.Migration.OffsetsCollection, targetClient)
+	err = commons.PrepareOffsetsCollection(ctx, r.Migration.OffsetsCollection, targetClient)
```

Comment:
this PR is not just about adding the tests - perhaps this change should land in some other PR?

Reply:
This is only a rename for consistency.